apps/site/lib/site_web/views/schedule_view.ex

```diff
@@ -54,6 +54,8 @@ defmodule SiteWeb.ScheduleView do
   @spec template_for_tab(String.t()) :: String.t()
   @doc "Returns the template for the selected tab."
   def template_for_tab(tab_name)
+  # Is trip-view ever rendered?
```

Comment:
Oh hey! This is the Schedule tab that we are about to remove this March. So it is rendered, but not for much longer!